scripts/add-login-issue.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+echo "Creating custom issue file for $1"
+
+echo "Welcome to the Clear Linux live image!
+
+ * Documentation:     https://clearlinux.org/documentation/clear-linux
+ * Community Support: https://clearlinux.org/community
+
+To install Clear Linux onto this sytem please login as root,
+enter a new temporary password, and run clr-installer.
```

Comment:
Could we change from:
`enter a new temporary password, and run clr-installer.`
to:
`enter a new temporary password, and run:\n
clr-installer`